v2/cmd/wails/main.go

```diff
@@ -40,7 +40,7 @@ func printBulletPoint(text string, args ...any) {
 	if err != nil {
 		fatal(err.Error())
 	}
-	t = strings.Trim(t, "\n\r")
+	t = strings.Trim(t, "")
```

Comment:
Does `string.TrimSpace()` do the same thing?

Reply:
Can I just delete this line of code?

Reply:
Give it a go :-)

Reply:
Has been resolved.